app/assets/javascripts/components/projects/ProjectLoading.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const ProjectLoading = () => 
+  <div className="ProjectBoard-loading" data-id="project-loading">
+    <div className="ProjectBoard-loading-content">
```

Comment:
Okay, now you have two root classes: `ProjectBoard-loading` and `ProjectBoard-loading-content`.
That is not a good practice too 😢 
To avoid that or you extract `ProjectBoard-loading-content` to a new component or you will have to rename your classes to:
```
ProjectBoard-loading-content -> ProjectBoard-loading__content
ProjectBoard-loading-content__spinner -> ProjectBoard-loading__spinner
ProjectBoard-loading-content__text -> ProjectBoard-loading__text
```
🤔

Reply:
Oh, I will rename the classes. Create a new component just for spinner I think that is unnecessary because it will not be used in other place.